src/lib-voyager.tsx

```diff
@@ -77,8 +81,24 @@ export class Voyager {
    * @memberof Voyager
    */
   public setSpec(spec: Object) {
-    const data = "data";
-    this.data = spec[data];
+
+    const ajv = new Ajv({
```

Comment:
Hmm -- don't we already do this somewhere? 

I mean it's good that we do here but I wanna make sure that we don't run it twice.